### DISCUSSION TITLE
Type of timer alarm - one time or until canceled?

### DISCUSSION THREAD

**Discussion by christianlupus** [nextcloud/cookbook#1125] [Polls]
Hello,

during our work on #671, we were facing a general decision to be made:
**Should a timer alarm ring just once or should it ring until the user cancels/acknowledges the alarm?**

There are pros and cons for both sides. We would like to know what you as our user base thinks more suitable and what you would prefer.

**Comment by tchernobog** [comment]
Giving that you might have something in the oven, it's better if it keeps ringing for safety reasons. It might make sense to put a cap on it, though, like 5 minutes or so after which it stops ringing and only a notification remains.

**Comment by christianlupus** [comment]
@MarcelRobitaille there has not been much of answers in the 2 weeks so far. Currently, more people tend to use the more persistent alarm system. However, I would suggest writing it in a way such that the addition can be made to only ring once. Having that in mind, it should not take too much additional effort.

What do you think of this?
